docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
 
 You can now add override rules that use a regex matcher and overrides that match on field type.
 
+## Query inspector in Explore
+
+You can enjoy all the details query inspector gave you in dashboards now in explore as well. You can open query inspector tab with the button next to query history.
```

Comment:
Capitalize the name of the tab. Match the UI.

Reply:
Done